packages/store/addon/-private/ts-interfaces/minimum-serializer-interface.ts

```diff
@@ -139,7 +139,7 @@ interface Serializer {
    * @method normalizeResponse
    * @public
    * @param {Store} store The store service that initiated the request being normalized
-   * @param {ShimModelClass} schema An object with methods for accessing information about
+   * @param {ModelSchema} schema An object with methods for accessing information about
```

Comment:
If we don't publish types, how does this work out for the docs?

Reply:
the docs don't link these things